lib/TopStoriesFeed.jsm

```diff
@@ -147,9 +155,11 @@ this.TopStoriesFeed = class TopStoriesFeed {
     }
     if (stories && stories.length > 0 && this.storiesLastUpdated === 0) {
       this.updateSettings(data.stories.settings);
-      const rows = this.transform(stories);
-      this.stories = rows;
+      this.stories = this.rotate(this.transform(stories));
```

Comment:
Can you explain adding `rotate` here?

Reply:
Rotate checks if a story can be removed from the list and the next one moved forward, in most cases that's because a story has expired.

I added the rotate here to try to figure out why the spoc kept going into new positions, thinking it had to do with rotate and wondering why cache load didn't do a rotate and fetch did. Wondered if this was my bug. It didn't fix it, but still thought it was weird that cache didn't do it and fetch did. So I asked Christian about it.

If you use it when nothing is ready to expire, it shouldn't do anything.

I left it here because if we're loading the same stories, chances are better that something has expired, it felt harmless, and It also has the added benefit of if the user is offline, their stories can now refresh. If there is potential that this could cause an issue, or you want it tackled in anther patch, happy to revert it, as it shouldn't have any effect on the rest of the patch.